index.html

```diff
@@ -59,8 +59,37 @@
     </a>
 
     <!-- Hero -->
-    <section id="hero" class="section-pad-left">
-
+    <section id="hero">
+        <div id="hero-content">
+            <h1>BrickHack 8</h1>
+            <h3>March 5 - 6, 2022 | RIT Online Hackathon</h3>
+            <p>Sign up to be notified when applications open: </p>
+            <!-- MailChimp email form, copied from bh8 preregister -->
+            <div class="field-row-stacked">
+                <div id="mc_embed_signup">
+                    <form action="https://coderit.us11.list-manage.com/subscribe/post?u=122b09a8cef4c1f3888af8e40&amp;id=4c1af7f783" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" rel="noopener" novalidate>
+                        <div id="mc_embed_signup_scroll">
+                            <div class="mc-field-group">
+                                <div class="field-row">
+                                    <input type="email" value="" name="EMAIL" class="required email" id="mce-EMAIL" placeholder="name@example.com">
+                                    <button type="submit">SUBMIT</button>
+                                </div>
+                            </div>
+                            <div id="mce-responses" class="clear">
+                                <div class="response" id="mce-error-response" style="display:none"></div>
+                                <div class="response" id="mce-success-response" style="display:none"></div>
+                            </div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+                            <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_122b09a8cef4c1f3888af8e40_4c1af7f783" tabindex="-1" value=""></div>
+                        </div>
+                    </form>
+                </div>
+            </div>
+        </div>
+        <div id="hero-video">
+            <video width="750" autoplay muted loop>
+                <source src="./assets/bh8/videos/LandingLoop.mp4" type="video/mp4">
```

Comment:
I'd really consider uploading a higher quality video here; it looks fuzzy on a 16" laptop. If design can play around with cropping video margin tighter, (should be H.264 encoding), that might allow more resolution. We can most likely push 2MB if it leads to a better experience but that's just subjective; I wasn't able to find a good direction on size; GitHub loads a 1.74MB video on their homepage once ya scroll down.

Reply:
I also think that a larger video (or same literal size but no margins in the file itself) would help fill space on larger screens, as @peterkos mentioned in his general review notes.

Reply:
I wonder if gifs would actually be a better solution here?

Reply:
Maybe? If the quality and file sizes work out right, then it's worth a try

Reply:
We're swapping around the layout, so let's just see how it looks after

Reply:
:+1: video quality looks fine on my 1080p, 13.4" display